src/fri/fri_config.cairo

```diff
@@ -0,0 +1,70 @@
+use core::array::ArrayTrait;
+use core::option::OptionTrait;
+use core::array::SpanTrait;
+use core::traits::Into;
+use cairo_verifier::table_commitment::TableCommitmentConfig;
+use cairo_verifier::vector_commitment::{validate_vector_commitment, VectorCommitmentConfig};
+
+const MAX_LAST_LAYER_LOG_DEGREE_BOUND: u32 = 15;
+const MAX_FRI_LAYERS: u32 = 15;
+const MAX_FRI_STEP: u32 = 4;
+
+#[derive(Drop, Copy)]
+struct FriConfig {
+    // Log2 of the size of the input layer to FRI.
+    log_input_size: felt252,
+    // Number of layers in the FRI. Inner + last layer.
+    n_layers: felt252,
+    // Array of size n_layers - 1, each entry is a configuration of a table commitment for the
+    // corresponding inner layer.
+    inner_layers: Span<TableCommitmentConfig>,
+    // Array of size n_layers, each entry represents the FRI step size,
+    // i.e. the number of FRI-foldings between layer i and i+1.
+    fri_step_sizes: Span<felt252>,
+    log_last_layer_degree_bound: felt252,
+}
+
+fn fri_config_validate(
+    config: FriConfig, log_n_cosets: felt252, n_verifier_friendly_commitment_layers: felt252
+) -> felt252 {
+    let n_layers = config.n_layers.try_into().unwrap();
+    let log_last_layer_degree_bound = config.log_last_layer_degree_bound.try_into().unwrap();
```

Comment:
I would suggest explicitly typing `log_last_layer_degree_bound` and `n_layers` as `u32`

Reply:
FriConfig is given to the programme externally tbh i would leave it so try_into checks ranges on those numbers
This is part of validity checking right?

Reply:
I meant to just add explicit `log_last_layer_degree_bound: u32`, not change `FriConfig` struct